controllers/remove.js

```diff
@@ -11,18 +11,40 @@ var Script = require('../models/script').Script;
 var User = require('../models/user').User;
 var destroySessions = require('../libs/modifySessions').destroy;
 
+var formidable = require('formidable');
+var statusCodePage = require('../libs/templateHelpers').statusCodePage;
+
 // Simple controller to remove content and save it in the graveyard
 exports.rm = function (aReq, aRes, aNext) {
   var type = aReq.params[0];
   var path = aReq.params[1];
   var authedUser = aReq.session.user;
 
+  var form = null;
+  var reason = null;
+
+  if (!/multipart\/form-data/.test(aReq.headers['content-type'])) {
+    return aNext();
+  }
+
+  form = new formidable.IncomingForm();
+  form.parse(aReq, function (aErr, aFields) {
```

Comment:
Good to know... will get that after slumber too. Have another use case to get too. Thanks.